moon/docker/rover/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ros:melodic
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update && apt-get install -y python-pip python3-pip openssh-client vim less
+RUN python -m pip install pyzmq
+
+RUN /usr/bin/python3 -m pip install --user --no-cache-dir msgpack "opencv-python>3.4.2,<4" pyzmq
```

Comment:
Everything above this line I would suggest to add to some base image that would be built only once and everyone would download it. The motivation is the same as in subt - every build of the above can produce a different results. If something is failing for one dev and not for the other, it could be difference in packages installed. But here the risk might not be as big as in subt where we are using a lot of packages (ignition, ros, rtabmap and others).

Reply:
Yes, good point - this was actually copy & paste of the master that time. One possibility is to share the base image, but ... there could be extra clash between moon and subt??

Reply:
I am not sure if that would be a good idea, it contains a lot of stuff already and it will contain even more in the future. So far I am aware of any advantage it would bring.